### PR TITLE
remove service and env

### PR DESCRIPTION
**Pull request by juli1**
### What and why?

We remove the `service` and `env` as mandatory arguments for the `sarif` and `abom` command. We put a default value. This default value will then be hidden from the user. It can used for graphs but not going to be surfaced in the product.

### How?

Put a default value in the argument

### Test

Screenshot 2024-08-07 at 6 27 01 PM

### CODE REVIEW

**Review by modernplumbing — APPROVED** [review]

File: src/commands/sarif/upload.ts [hunk 1]
```diff
@@ -71,15 +71,6 @@ export class UploadSarifReportCommand extends Command {
   }
 
   public async execute() {
-    if (!this.service) {
-      this.service = process.env.DD_SERVICE
```
[the remark this answers is not in the document]
Comment (juli1) [reply src/commands/sarif/upload.ts:75]: At least for the `sarif` and `sbom`. Will document

File: src/commands/sbom/upload.ts [hunk 2]
```diff
@@ -52,22 +52,9 @@ export class UploadSbomCommand extends Command {
    * compliant with their schema and upload them to datadog.
    */
   public async execute() {
-    const service: string | undefined = this.service || process.env.DD_SERVICE
+    const service: string = this.service
```
[the remark this answers is not in the document]
Comment (juli1) [reply src/commands/sbom/upload.ts:55]: At least for the `sarif` and `sbom`. Will document

File: src/commands/sbom/README.md [hunk 3]
```diff
@@ -1,36 +1,37 @@
...
- - `DD_SERVICE`: the Datadog service you use (if `--service` not specified)
-
 
 ## Development
 
 When developing software, you can try with the following command:
 
-```bash
-DD_BETA_COMMANDS_ENABLED=1 yarn launch sbom upload --service <your-service> --env <your-environment> /path/to/sbom.json
+```bashyarn launch sbom upload --service <your-service> --env <your-environment> /path/to/sbom.json
```
Comment (juan-fernandez) [inline src/commands/sbom/README.md:34]: I don't think this is rendered correctly
```suggestion
```bash
yarn launch sbom upload --service <your-service> --env <your-environment> /path/to/sbom.json
```

**Review by juan-fernandez — APPROVED** [review]

File: src/commands/sarif/README.md [hunk 4]
```diff
@@ -20,10 +20,10 @@ For example:
 datadog-ci sarif upload --service my-service --tags key1:value1 --tags key2:value2 sarif-reports/go-reports sarif-reports/java-reports sarif-report/single-report.sarif
 ```
 
-- The positional arguments are the directories or file paths in which the SARIF reports are located. If you pass a folder, the CLI looks for all `.sarif` files in it.
-- `--service` (default: `DD_SERVICE` env var) should be set to the name of the service you're uploading SARIF reports for.
+The positional arguments are the directories or file paths in which the SARIF reports are located. If you pass a folder, the CLI looks for all `.sarif` files in it.
+- `--service` should be set to the name of the service you're uploading SARIF reports from.
```
Comment (amaanq) [inline src/commands/sarif/README.md:24]:
```suggestion
- `--service` should be set to the name of the service you're uploading SARIF reports to.
```

**Review by amaanq — APPROVED** [review]

**Pull request merged**